CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
 - Slight optimization so that offerings and purchaserInfo are returned faster if they're cached.
     https://github.com/RevenueCat/purchases-ios/pull/433
     https://github.com/RevenueCat/purchases-ios/issues/401
-- Revamped logging strings, makes log messages from `Purchases` easier to spot and understand.
+- Revamped logging strings, makes log messages from `Purchases` easier to spot and understand. Deprecated `RCReceiptInUseByOtherSubscriberError`
+to use `RCReceiptAlreadyInUseError`.
```

Comment:
let's do the update as a separate line, to make it very clear, and update the text to reflect that it's not a deprecation, it's a removal. 

how about: 

```markdown
- Revamped logging strings, makes log messages from `Purchases` easier to spot and understand. 
- Removed `RCReceiptInUseByOtherSubscriberError`, replaced by `RCReceiptAlreadyInUseError`.
```

Reply:
on second thought, adding another bullet point right next to the previous one would break the format, so it's fine to have them both on the same one. But we should call it removed I think since it represents what happened a bit more closely

Reply:
it looks like your 3.10.2. PR will add this update, should I just close out this PR?

Reply:
yeah, feel free to close it and I'll integrate the changes back into `develop` once the other one gets merged. sorry for the noise

Reply:
sounds good. No worries, thanks for handling a new release.